evaluation/benchmarks/swe_bench/run_infer.py

```diff
@@ -56,21 +55,9 @@ def _get_swebench_workspace_dir_name(instance: pd.Series) -> str:
 def get_instruction(instance: pd.Series, metadata: EvalMetadata):
     workspace_dir_name = _get_swebench_workspace_dir_name(instance)
     # Prepare instruction
-    if metadata.agent_class == 'CodeActSWEAgent':
-        instruction = (
-            'We are currently solving the following issue within our repository. Here is the issue text:\n'
-            '--- BEGIN ISSUE ---\n'
-            f'{instance.problem_statement}\n'
-            '--- END ISSUE ---\n\n'
-        )
-        if USE_HINT_TEXT and instance.hints_text:
-            instruction += (
-                f'--- BEGIN HINTS ---\n{instance.hints_text}\n--- END HINTS ---\n'
-            )
-        instruction += CODEACT_SWE_PROMPT.format(workspace_dir_name=workspace_dir_name)
-    else:
-        # Instruction based on Anthropic's official trajectory
-        # https://github.com/eschluntz/swe-bench-experiments/tree/main/evaluation/verified/20241022_tools_claude-3-5-sonnet-updated/trajs
+
+    # Instruction based on Anthropic's official trajectory
+    # https://github.com/eschluntz/swe-bench-experiments/tree/main/evaluation/verified/20241022_tools_claude-3-5-sonnet-updated/trajs
```

Comment:
Do we need to fix indentation here?

Reply:
I think it was in an else, and now it's not

Reply:
double checking

Reply:
Openhands fix success summary


The AI has confirmed that the indentation issue has been fixed by properly indenting the `instruction = (...)` statement with 4 spaces and its continuation lines with 8 spaces, which follows Python's standard indentation style. This addresses the feedback about checking if the code was previously in an else block and ensuring proper indentation. The change maintains Python's code formatting standards while preserving the logical structure of the code.

Reply:
So cool that it answered here!